crates/generated_parser/src/ast_builder.rs

```diff
@@ -4019,11 +4024,15 @@ impl<'alloc> AstBuilder<'alloc> {
         name: Option<arena::Box<'alloc, BindingIdentifier>>,
         mut tail: arena::Box<'alloc, ClassExpression<'alloc>>,
     ) -> arena::Box<'alloc, Expression<'alloc>> {
+        let offset = class_token.loc.start;
         let index = self
             .context_metadata
-            .find_first_binding(class_token.loc.start);
+            .find_first_binding(offset);
         self.context_metadata.pop_bindings_from(index);
 
+        let label_index = self.context_metadata.find_first_label(offset);
+        self.context_metadata.pop_labels_from(label_index);
```

Comment:
there shouldn't be any unhandled labels at this point, given that unhandled labels should be caught by methods.